src/components/overview/OverviewMain.jsx

```diff
@@ -134,11 +134,11 @@ class overviewMain extends React.Component {
   render() {
     return (
       <div>
-        <div class="level">
-          <p class="level-left">
+        <div class="columns">
```

Comment:
Interested in your thought process of switching from level to columns. Did you notice limitations of using Bulma's level styling?

Reply:
still trying to figure things out! it's hard to get used to